packages/react-chat/src/utils/config.ts

```diff
@@ -15,17 +15,50 @@ const tryDecodeURIComponent = (str: string) => {
   }
 };
 
+// eslint-disable-next-line xss/no-mixed-html
+const sanitizeRenderOptions = (renderOptions: any): Partial<ChatConfig['render']> => {
+  if (!isObject(renderOptions)) return { mode: RenderMode.BUBBLE };
+
+  const { mode, target } = renderOptions;
```

Comment:
could just do 
```suggestion
  const { mode, target = document.getElementById('voiceflow-chat-frame') } = renderOptions;
```

and remove the conditional block below for `fallbackHost`